tutorials/notebooks/vo/conesearch.ipynb

```diff
@@ -141,7 +137,7 @@
    "source": [
     "# Now use your favorite text editor to open the my_result.tex file.\n",
     "# For example:\n",
-    "!more my_result.tex"
+    "!cat my_result.tex"
```

Comment:
Just curious, does your machine not have `more`?

Reply:
On my machine, `more` never lets the cell finish executing, so subsequent cells never run.

Reply:
:scream: ok